axelor-human-resource/src/main/resources/i18n/messages_fr.csv

```diff
@@ -37,6 +37,7 @@
 "Amendment type","Type d'avenant",,
 "Amendment types","Type d'avenants",,
 "Amount","Montant",,
+"Amount limit","Montant limit",,

```

Comment:
The translation should be "Limite de montant", it was my mistake, I gave a bad translation.